src/main/java/com/google/javascript/cl2dts/DeclarationGenerator.java

```diff
@@ -829,7 +829,13 @@ private void visitObjectType(ObjectType type, ObjectType prototype, Boolean proc
     }
 
     private void visitProperties(ObjectType objType, boolean isStatic) {
+      visitProperties(objType, isStatic, null);
```

Comment:
Why not always pass `Collections.emptySet()` here, so that you don't have to handle `null` below? It's usually better to avoid `null` and favour empty collections instead (fewer code paths, less pondering on nullness, etc).